src/main/java/org/heigit/ohsome/ohsomeapi/oshdb/ExtractMetadata.java

```diff
@@ -13,6 +14,7 @@
   public static Geometry dataPoly = null;
   public static JsonNode dataPolyJson = null;
   public static int replicationSequenceNumber;
+  public static double timeOutInSeconds = ProcessingData.getTimeout();
```

Comment:
The parameter is called "timeout", the method in the class ProcessingData.java is called getTimeout(), but here you name the variable "timeOut..." with a big "O". This does not match with the current writing of the word "timeout" in the other parts of the code.